packages/@aws-cdk/pipelines/README.md

```diff
@@ -38,10 +55,38 @@ will work, see the section **CDK Environment Bootstrapping** below).
   * to this table in its properties.
   */
 import { DatabaseStack, ComputeStack } from '../lib/my-stacks';
-
 import { Construct, Stage, Stack, StackProps, StageProps } from '@aws-cdk/core';
-import { CdkPipeline } from '@aws-cdk/pipelines';
-import * as codepipeline from '@aws-cdk/aws-codepipeline';
+import { CodePipeline, CodePipelineSource, ScriptStep } from '@aws-cdk/pipelines';
+
+/**
+ * Stack to hold the pipeline
+ */
+class MyPipelineStack extends Stack {
+  constructor(scope: Construct, id: string, props?: StackProps) {
+    super(scope, id, props);
+
+    const pipeline = new CodePipeline(this, 'Pipeline', {
+      synth: new ScriptStep('Synth', {
+        // Will use a SecretsManager secret named 'github-token' by default
```

Comment:
Is there any reason this does not use GitHubv2 access? Token-based auth is deprecated afaik.

Reply:
Gimme a link please? This is the first I'm hearing of this

Reply:
![image](https://user-images.githubusercontent.com/1287829/125636959-18e48915-dd9f-4759-be11-800bc655feac.png)